samples/dlp_inspect_image_file.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright 2018, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+// [START dlp_inspect_image_file]
+// Imports the Google Cloud Data Loss Prevention library
+const DLP = require('@google-cloud/dlp');
+
+// Import other required libraries
+const fs = require('fs');
+
+export async function inspectImageFile(
+  projectId = 'YOUR_PROJECT_ID',
```

Comment:
We should never require the `projectId`. Best practice is to leverage the metadata server present on all GCP runtimes.

Reply:
DLP is somewhat special as the request needs the project ID, users should not pass it in though but use `dlp.getProjectId()`, https://github.com/googleapis/nodejs-dlp/blob/master/src/v2/dlp_service_client.js#L268.

Reply:
It's not actually that special, quite a few APIs require project ID in path parameters to specify the resource owner for billing (different from the inferred project ID used for authentication). Current guidance from the sample working group is to make project ID explicit in the case where it's a required value in the request body. For almost all samples however, the project ID used for authentication should be inferred from the environment (hence the use of `const dlp = new DLP.DlpServiceClient();` in the sample)

Reply:
OK